support/deploy/run.sh

```diff
@@ -38,7 +38,7 @@ case $2 in
   "aura")
     host="aura.ethereum.nodes.augur.net"
     port="80"
-    privateKey="fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a"
+    privateKey="47f49c399482f73143cadeb2db8938d3f249578bdc64cdcda4ecf1ee535a5c91"
```

Comment:
Why are we changing the Aura private key?  The private key currently used by Auru is the same one used by Clique and Instantseal, and it is the one that the node should have setup to be seeded with all of the money.

Reply:
Currently the clique key is different and does not have ether.  It is actually the clique key that should be replaced with the aura key, not the other way around!

Reply:
Actually it turns out, @tinybike that that key file is wrong and it should be the one located in the "keys" subdirectory, and clique/aura have the same key.